Doc/library/xml.etree.elementtree.rst

```diff
@@ -638,6 +638,83 @@ Functions
    :class:`Element` instance and a dictionary.
 
 
+.. _elementtree-xinclude:
+
+XInclude support
+----------------
+
+This module provides limited support for
+`XInclude directives <https://www.w3.org/TR/xinclude/>`_ , via the **ElementInclude** helper module.  This module can be used to insert subtrees and text strings into element trees, based on information in the tree.
+
+Example
+^^^^^^^
+
+Here's an example that demonstrates use of the XInclude module. To include a XML document in the current document, use the ``{http://www.w3.org/2001/XInclude}include`` element and set the **parse** attribute to ``“xml”``, and use the **href** attribute to specify the document to include.::
```

Comment:
"an XML document". The pronunciation of XML ("ex-em-ell") starts with a vowel.

Reply:
Thanks @scoder for the valuable feedback. I'll update them.